rust/config/mainnet2/mainnet2_config.json

```diff
@@ -152,6 +152,25 @@
       "index": {
         "from": "2595747"
       }
+    },
+    "gnosis": {
+      "name": "gnosis",
+      "domain": "100",
+      "addresses": {
+        "mailbox": "0x35231d4c2D8B8ADcB5617A638A0c4548684c7C70",
+        "interchainGasPaymaster": "0x6cA0B6D22da47f091B7613223cD4BB03a2d77918",
+        "validatorAnnounce": "0x9bBdef63594D5FFc2f370Fe52115DdFFe97Bc524"
+      },
+      "signer": null,
+      "protocol": "ethereum",
+      "finalityBlocks": "14",
+      "connection": {
+        "type": "http",
+        "url": ""
+      },
+      "index": {
+        "from": "26149361"
```

Comment:
Just flagging that this may have been re-written on re-deploy and we should use the original value

Reply:
This is the value for the deployment of the contracts on gnosis, so there would be no original value right?